utilities.c

```diff
@@ -435,3 +435,76 @@ uint64_t iio_read_counter_us(void)
 
 	return value;
 }
+
+int iio_get_range_from_available(const char *input, double *min, double *step, double *max)
+{
+	double lmin, lstep, lmax;
+	char extra;
+	int n;
+
+	if (!input)
+		return -EINVAL;
+
+	// Expect format: [min step max]
+	#if defined(_MSC_VER)
+    n = iio_sscanf(input, " [ %lf %lf %lf %c", &lmin, &lstep, &lmax, &extra, (unsigned int)sizeof(extra));
+#else
+    n = iio_sscanf(input, " [ %lf %lf %lf %c", &lmin, &lstep, &lmax, &extra);
+#endif
+
```

Comment:
this looks like white space issues? The #if should be alighned with the #else/#endif?